> Haven't tested this. Can we count on the tests to cover it?

Pretty sure tests will cover it.
The CI did catch a small PEP8 mismatch. Can you fix it, and I'll merge the change :)

`src/time_travel/event_pool.py:48:62: W291 trailing whitespace`

--- a/src/time_travel/event_pool.py
+++ b/src/time_travel/event_pool.py
@@ -45,7 +45,7 @@ class EventPool(object):
             out = []
 
             for fd, event_set in _ts_dict.items():
-                out_events = {event for event in event_set if 
+                out_events = {event for event in event_set if
                               predicate(fd, event)}
 
                 if out_events:

